Linux_template_config_instructions.txt

```diff
@@ -12,7 +12,7 @@ vi /etc/ssh/sshd_config
 kill -HUP `pidof sshd`
 apt-get install -y git
 ssh-keygen
-Add key to GitHub
+#am sters linia asta
```

Comment:
Always use English when coding -- for branch names or comments or commit messages